documentation/man/kw.rst

```diff
@@ -0,0 +1,217 @@
+files. Additionally, there is many other features that **kw** provides but that
+not fit in the previous section; here you can find all the other elements.
+
+g, configm [*--save NAME* [*-d DESCRIPTION*][-f]]|[*--ls*]
+----------------------------------------------------------
+
+The 'configm' option it is the application that manages the '.config' files for
+users. In summary, it provides operations for save, load, removes, and list
+'.config' previously saved by the user. See the current options:
+
+1.  --save NAME [-d DESCRIPTION] [-f]: The save option seeks in the current
+directory for a '.config' file to be added under the management of kw.  The
+save option expects a name to be used as an alias for the target config file.
+If we have a local .config and a valid name, kw saves the configuration file.
+Additionally, users can add a description by using *-d* flag. Finally, if the
+user tries to add the same name twice kw will warn about it; the '-f' will
+suppress this message.
+
+2. --ls list all the config files available.
+
+v, vars
+-------
+Show variable status used by **kw**.
+
+bi
+--
+Build and install a new module version. It is a combination of **build** and
+**install**.
+
+h, help
+-------
+Show basic help.
+
+ABOUT kworflow.config
+=====================
+**kw** uses a configuration file named **kworkflow.config**. We have two ways
+of using this configuration file: (1) the globals and (2) the local. The global
+**kworkflow.config** is kept with the **kw** code and provides the overall
+behavior for **kw**. The local **kworkflow.config** has precedence over the
+global, you can have one **kworkflow.config** per project with specific
+configurations. In this section, we describe the field available by **kw**.
+
+ssh_ip=IP
+---------
+Set the IP to be used by the ssh. By default **kw** uses **localhost**.
+
+ssh_port=PORT
+-------------
+Set the SSH port. By default **kw** uses 2222.
+
+arch=ARCHITECTURE
+-----------------
+Allow you to Specify the default architecture used by **kw**. By default,
+**kw** uses x86_64.
+
+virtualizer=VIRTUALIZER
+-----------------------
+Defines the virtualization tool that should be used by kw. Currently, we only
+support QEMU, as a result, te default is **qemu-system-x86_64**
+
+mount_point=PATH
+----------------
+Defines the kw mount point, this directory to be used by libguestfs during the
+mount/umount operation of a VM. By default, **kw** uses /home/$USER/p/mount.
+
+qemu_hw_options=OPTIONS
+-----------------------
+Sets QEMU options. By default, **kw** uses
+**-enable-kvm -daemonize -smp 2 -m 1024**
+
+qemu_net_options=OPTIONS
+------------------------
+Defines the network configuration. By default, **kw** uses
+**-net nic -net user,hostfwd=tcp::2222-:22,smb=/home/USER**
+
+qemu_path_image=PATH
+--------------------
+Specify the VM image path. By default, **kw** uses
+**/home/USERKW/p/virty.qcow2**
+
+alert=[vs|s|v|n]
+----------------
+Default alert options, you have:
```

Comment:
I think we should give the user a little more background of this variable. We should say that the value specified here will be used by kw with commands that use alert notification when the user doesn't specify a --alert= option. Also, we lost the description of the --alert= option in the transition from README to this man page. We should say, somewhere, what this option does and which commands accept it.